Source/SwiftLintFramework/Rules/TodoRule.swift

```diff
@@ -41,14 +41,49 @@ public struct TodoRule: ConfigurationProviderRule {
         ]
     )
 
+    // swiftlint:disable conditional_binding_cascade
```

Comment:
this rule was removed, so I'd rather not add this, only to have to remember to remove it at some point down the line.